test/packages/caver.rpc.js

```diff
@@ -939,8 +1124,53 @@ describe('caver.rpc.governance', () => {
         }).timeout(100000)
     })
 
+    context('caver.rpc.governance.getChainConfigAt', () => {
+        it('CAVERJS-UNIT-RPC-015: should return the latest chain configuration', async () => {
+            const govRPCStub = sandbox
+                .stub(caver.rpc.governance.getChainConfigAt.method.requestManager, 'send')
+                .callsFake((payload, sendTxCallback) => {
+                    expect(payload.method).to.equal('governance_chainConfigAt')
+                    expect(payload.params.length).to.equal(caver.rpc.governance.getChainConfigAt.method.params)
+                    const ret = {
+                        chainId: 1001,
+                        istanbulCompatibleBlock: 0,
+                        londonCompatibleBlock: 0,
+                        ethTxTypeCompatibleBlock: 0,
+                        magmaCompatibleBlock: 1,
+                        deriveShaImpl: 2,
+                        governance: {
+                            governanceMode: 'ballot',
+                            governingNode: '0xe733cb4d279da696f30d470f8c04decb54fcb0d2',
+                            reward: {
+                                deferredTxFee: true,
+                                minimumStake: 5000000,
+                                mintingAmount: 9600000000000000000,
+                                proposerUpdateInterval: 3600,
+                                ratio: '34/54/12',
```

Comment:
Please check the kip82Ratio with klaytn 1.10.

Reply:
Thanks, I updated ratio info.